Makefile

```
@@ -6,21 +6,28 @@ build:
 	bin/docker-prepare && \
 	docker-compose -f docker-compose.test.yml build
 
-run:
+test_variant_app:
 	cd $(APP_NAME) && \
 	docker-compose -f docker-compose.test.yml run test
 
-base_spec = spec/base/**/*_spec.rb,spec/addons/docker/**/*_spec.rb
-test:
+base_addon_spec = spec/addons/base/**/*_spec.rb
+web_addon_spec = spec/addons/variants/web/**/*_spec.rb
+api_addon_spec = spec/addons/variants/api/**/*_spec.rb
+
+base_spec = spec/base/**/*_spec.rb
+web_spec = spec/variants/web/**/*_spec.rb
+api_spec = spec/variants/api/**/*_spec.rb
+
+test_template:
 	cd $(APP_NAME) && \
 	docker-compose -f docker-compose.test.yml up --detach db redis && \
 	docker-compose -f docker-compose.test.yml run --detach test bin/start.sh && \
 	cd ../.template && \
 	bundle install; \
 	if [ $(VARIANT) = web ]; then \
-	  bundle exec rspec --pattern="$(base_spec),spec/variants/web/**/*_spec.rb"; \
+		bundle exec rspec --pattern="$(base_addon_spec),${base_spec},$(web_addon_spec),$(web_spec)"; \
```

Is it intended to have a mix of `()` and `{}` 🤔 Otr should it be:

```suggestion
		bundle exec rspec --pattern="${base_addon_spec},${base_spec},${web_addon_spec},${web_spec}"; \
```

---

Oh my bad, will update it :)

---

Fixed in https://github.com/nimblehq/rails-templates/pull/248/commits/c87b51926b9f1d643ccb7473b4970c56dbf26aa3 @olivierobert